contributed_definitions/nyaml/NXellipsometry.yaml

```diff
@@ -201,39 +150,25 @@ NXellipsometry(NXopt):
             doc: |
               Specify the maximum value of revolutions of the rotating element
               for each measurement.
-        spectrometer(NXmonochromator):
-          exists: optional
-          doc: |
-            The spectroscope element of the ellipsometer before the detector,
-            but often integrated to form one closed unit. Information on the
-            dispersive element can be specified in the subfield GRATING. Note
-            that different gratings might be used for different wavelength
-            ranges. The dispersion of the grating for each wavelength range can
-            be stored in grating_dispersion.
     (NXsample):
       backside_roughness(NX_BOOLEAN):
+        exists: recommended
         doc: |
           Was the backside of the sample roughened? Relevant for infrared
           ellipsometry.
-    data_collection(NXprocess):
+    (NXdata):
```

Comment:
I still have problem digesting this usage of NXdata, which should be a plottable view of the data, but not the data itself...

Reply:
I just found out in discussion with sandor, what the initial intent of data_collection(NXprocess) was.

I updated the NXellipsometry definition to include this in "data_colletion" under "NXdata". This may need to be specialized later for NXopt as well.

I intended to save here the data, which is not referred as raw-data (raw data is located in the detector). As well, the default plot shall be given (see (NXdata) in NXopt).

Reply:
Data Collectino may have been executed by a specific software which may have even did some data processing, and as such NXprocess is a better concept than a simply a plotable NXdata. This we can discuss on the OPT Workshop and make the update according to the conclusions from there.